l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -942,6 +943,8 @@ def invoiceCreate(
         self._addGlobalDiscount(
             invoice_id, FatturaBody.DatiGenerali.DatiGeneraliDocumento)
 
+        self.set_roundings(FatturaBody, invoice)
```

Comment:
Da aggiungere anche nel wizard di link

Reply:
Ok

Reply:
Non mi è chiaro questo caso: in odoo viene creata la fattura bozza, tipicamente da un ordine di acquisto; il sistema riceve la fattura XML, con arrotonamento non previsto; nel processo di "collegamento", il sistema aggiunge in autonomia le righe per l'arrotondamento.
Giusto?

PS: non sarebbe male aggiungere il test anche per questo caso

Reply:
@eLBati  Si esatto

Reply:
OK.
Il test di questo caso lo rimandiamo?

Reply:
@eLBati Se possibile per il momento rimanderei, poi farò una PR apposita

Reply:
@eLBati @SimoRubi Stavo ripensando alla ``self.set_roundings()`` da fare sul wizard di link, e mi stà venendo il dubbio che non sia proprio da fare! La ``self.set_roundings()`` aggiunge la riga per gestire gli arrotondamenti attivi/passivi ad una fattura già esistente, inserita dall'utente, secondo me non dovremmo proprio cambiare la registrazione. Ok per aggiungere le info della fattura elettronica, ma cambiarla mi sembra che non vada proprio bene. Che dite?

Reply:
Essendo una domanda più sul funzionale, mi astengo! :confounded:

Reply:
@robyf70 sono d'accordo

Reply:
Ok! Allora reverto la parte del wizard di link e poi vediamo dopo che fare